wizard_builder/forms.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, *args, **kwargs):
         for item in self.items:
             question_id = 'question_%s' % item.pk
             self.fields[question_id] = item.make_field()
-            self.fields[question_id].help_text = mark_safe(item.descriptive_text + self.fields[question_id].help_text)
+            self.fields[question_id].help_text = mark_safe(item.example + '<br/><br/>' + item.descriptive_text + self.fields[question_id].help_text)
```

Comment:
And yea I agree that doesn't seem like the right order. It was the currently existing order though. Should I change it from that?